### PR TITLE
feat(vscode): Port to VSCode

### PR DESCRIPTION
**Pull request by GoForceX**
Add VSCode version and corresponding guide to enhance usability for newcomers.

Providing rendered version below: 
[template.pdf](https://github.com/user-attachments/files/16965746/template.pdf)

### CODE REVIEW

**Comment by ligen131** [comment]
Preview: 
image

May have issues with dark themes
image

Thanks for your contribution!

**Comment by ligen131** [comment]
> There's no need to support a dark version for the cv style 🤔

Yes, you're right. But in the VS Code preview, it is displayed according to the IDE theme.

**Comment by YiNNx** [comment]
> Should I update my version to make it looks like the same?

Feel free to keep the changes!

**Review by YiNNx — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -105,3 +111,14 @@
...
 ```
+
+#### VSCode
+
+修改 `lapis-cv.css` 即可。具体样式可参考上述 Typora 部分。
+
+但页边距需要在 VSCode 设置中修改：
+
+1. 打开 VSCode 设置，选中 Workspace 标签页。
+2. 搜索 Markdown-pdf › Margin，并修改四边边距。<br>
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply README.md]: Choose method 2 to be the main one, and preserved method 1 as an alternative method.

File: README.md [hunk 2]
```diff
@@ -48,6 +48,12 @@
     
 3. 打开模版文件 [template.md](typora/template.md) 进行内容编辑，文件 - 导出 - PDF 即可导出 PDF 文件。
 
+#### VSCode
+
+1. 下载 [Release](https://github.com/BingyanStudio/LapisCV/releases/latest) 并解压至某一文件夹。
+2. 在 VSCode 中打开该文件夹，并安装插件 [Markdown PDF](https://marketplace.visualstudio.com/items?itemName=yzane.markdown-pdf)。
+3. 打开模版文件 [template.md](vscode/template.md) 进行内容编辑，右键选择 Markdown PDF - Export (pdf) 即可导出 PDF 文件。
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply README.md:55]: Leave vscode/template.md as it is to ensure relative path relationship.

File: vscode/lapis-cv.css [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    color: var(--text-color);
+  }
+}
+
+@media screen {
+  body {
+    box-shadow: 0 0 4mm 0 var(--shadow-color);
+    padding-left: 20mm;
+    padding-right: 20mm;
+    padding-top: 12mm;
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply vscode/lapis-cv.css:70]: Added padding-bottom: 12mm;

File: vscode/lapis-cv.css [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+  z-index: 9999;
+  float: right;
+  object-fit: cover;
+  overflow: hidden;
+
+  width: var(--avatar-width);
+  height: var(--avatar-width);
+  border-radius: 50%;
+  border: 2px solid var(--border-color);
+  box-shadow: 0 0 0 3mm var(--bg-color);
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply vscode/lapis-cv.css:196]: @media screen {
  img[alt="avatar"] {
    box-shadow: 0 0 0 3mm var(--vscode-editor-background);
  }
}

File: vscode/lapis-cv.css [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+}
+
+@media print {
+  body {
+    color: var(--text-color);
+  }
+}
+
+@media screen {
+  body {
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply vscode/lapis-cv.css:66]: Done.

File: vscode/lapis-cv.css [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+
+p:has(img[alt="avatar"]) {
+  height: 0;
+  text-align: center;
+}
+
+@media screen {
+  img[alt="avatar"] {
+    top: 1mm;
+    right: 0;
```
[the remark this answers is not in the document]
Comment (GoForceX) [reply vscode/lapis-cv.css:208]:
```css
@media screen {
  img[alt="avatar"] {
    position: relative;
    top: 0;
    right: 0;
  }
}
```

**Comment by GoForceX** [comment]
Preview: 

image
image

**Review by YiNNx — APPROVED** [review]

**Pull request merged**